src/GraphQL/ParseGraphQLServer.js

```diff
@@ -98,14 +98,13 @@ class ParseGraphQLServer {
       (_req, res) => {
         res.setHeader('Content-Type', 'text/html');
         res.write(
-          renderPlaygroundPage({
+          renderGraphiQL({
```

Comment:
I'd probably address this on a separate pr

Reply:
The playground is now not maintained anymore. This is why yoga exposes the `renderGraphiQL`.

We also already have graphiQL on the parse dashboard: https://github.com/parse-community/parse-dashboard/blob/e43ba0bf18aa2e951b6dd22fcaf73e961cd0c780/package.json#L47

Better DX since developers will have the same GQL IDE on both projects.

ref: https://github.com/graphql/graphql-playground/issues/1366#issuecomment-1062088978